### ISSUE TITLE
Allow for the Changing of Button Item Image Sizes?

### ISSUE THREAD

**Issue by NMoraton** [uias/Tabman#144]
Was wondering if it was possible for us to change the size of the images added to the TabBar? They seem to currently have a set height and width and I was curious if that was settable?

**Comment by NMoraton** [comment]
Or do I need to create a custom TabBar?

**Comment by msaps** [comment]
Hi, 

The image sizes are currently hardcoded and not configurable. I took a similar approach to `UITabBar`, so the sizes are currently 25x25pt when image only, and 20x20pt when using text and images in the bar item. 

My only reservation with making them configurable is being able to reliably display/layout the bar correctly with dynamic image sizing. A custom tab bar might be a good solution though, so you can get exactly the layout you want without having to compromise 👍

**Comment by NMoraton** [comment]
Hey msaps - Thanks for the quick response!

So right now I have just modified the source files in Tabman to update the size of the image to 35x35, so they are still static in the sense that they will always be 35x35 (I know this isn't advisable, but I wanted to see how it would look, and the new size is perfect!). I just wanted them to be a bit bigger and more viewable to the user. 

This is the line of code that I have updated in the TabmanButtonBar.swift file:
`static let itemImageSize: CGSize = CGSize(width: 35.0, height: 35.0)`

I should also note that because I am working with Swift 3.0 I am using the 0.8.3 version of Tabman.

So I was just wondering if a custom tab bar was the way to go, or if there was an easier solution?

Thanks!
